<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Search_Project</class>
 <widget class="QDialog" name="Form_Search_Project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rechercher...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="text">
        <string>Rechercher</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Recherche de...</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QCheckBox" name="tests_search_indic">
        <property name="text">
         <string>dans les scénarios</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="requirements_search_indic">
        <property name="text">
         <string>dans les exigences du projet</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Texte à chercher :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_text"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="case_sensitive_indic">
        <property name="text">
         <string>Respecter la casse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="tests_cases_search_indic">
        <property name="text">
         <string>dans les cas de tests</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QCheckBox" name="category_indic">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Catégorie</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="priority_indic">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Priorité</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comparison">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Nulle</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Faible</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Moyenne</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Elevée</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Critique</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="test_priority_level">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="test_category">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="text_search_indic">
     <property name="text">
      <string>Recherche textuelle</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="tests_from_current_project_version">
     <property name="text">
      <string>Recherche des tests de la version courante</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="requirements_from_current_project_version">
     <property name="text">
      <string>Recherche des exigences de la version courante</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QRadioButton" name="not_covered_requirements_search_indic">
     <property name="text">
      <string>Recherche des exigences non couvertes</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>text_search_indic</tabstop>
  <tabstop>search_text</tabstop>
  <tabstop>case_sensitive_indic</tabstop>
  <tabstop>tests_search_indic</tabstop>
  <tabstop>tests_cases_search_indic</tabstop>
  <tabstop>requirements_search_indic</tabstop>
  <tabstop>tests_from_current_project_version</tabstop>
  <tabstop>requirements_from_current_project_version</tabstop>
  <tabstop>not_covered_requirements_search_indic</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>search_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
